classes/managers/manage_genres.rb

```diff
@@ -1,46 +1,34 @@
 require_relative '../genre'
+require_relative '../io/file_writer'
+require_relative '../io/file_reader'
 
 class ManageGenres
-  def initialize(items)
-    @items = items
+  def initialize(labels)
+    @labels = labels
+    @file_writer = FileWriter.new('labels.json')
+    @file_reader = FileReader.new('labels.json')
```

Comment:
Wouldn't be better to rename the `labels` with `genres` because it's all related to genres?